src/scripts/main.js

```diff
@@ -1 +1,86 @@
 'use strict';
+
+let isPromise1Clicked = false;
+
+// eslint-disable-next-line prefer-const
+let isLeftMbClicked = false;
+// eslint-disable-next-line prefer-const
+let isRightMbClicked = false;
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    isPromise1Clicked = true;
+    resolve();
+  });
+
+  if (isPromise1Clicked === false) {
+    setTimeout(reject, 3000);
+  }
```

Comment:
The rejection logic for `promise1` might not work as intended. If the click event occurs after the 3-second timeout, the promise will not reject. Consider using a more reliable approach to handle the rejection, such as setting a timeout that checks if the promise has already been resolved.